lib/utils/bootstrap-ui-package.ts

```diff
@@ -0,0 +1,65 @@
+import {sync as commandExistsSync} from 'command-exists';
+import {join} from 'path';
+import {SpawnSyncStrict} from './spawn-sync-strict';
+
+const defaultSpawnOptions = {stdio: 'inherit', encoding: 'utf-8'} as const;
+
+export function bootstrapUIPackage(answers: any, context) {
```

Comment:
`answers` and `context` need type

Reply:
Documenting the function with a description would be good too